ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -947,12 +947,14 @@ periodics:
                   curl -L "${tar_url}" | tar xf - -C /srv/repo/ && \\
                   rm -rf /etc/yum.repos.d && \\
                   ostree --repo=/srv/repo checkout "${ostree_ref}" --subpath /usr/etc/yum.repos.d --user-mode /etc/yum.repos.d && \\
+                  ostree --repo=/srv/repo cat "${ostree_ref}" /usr/lib/os-release > /etc/os-release && \\
+                  source /etc/os-release && \\
```

Comment:
Hmm...a bit odd/dangerous to blow away the container's `/etc/os-release`.  Why not do:
```
                  ostree --repo=/srv/repo cat "${ostree_ref}" /usr/lib/os-release > target-os-release && \\
                  source target-os-release && \\
```
?

Reply:
Good idea, copied that to `/tmp/os-release` instead